common/app/routes/Hikes/redux/index.js

```diff
@@ -1,6 +1,6 @@
-export actions from './actions';
-export reducer from './reducer';
-export types from './types';
+export * from './actions';
```

Comment:
This should be `export * as actions from './actions'`

see: https://github.com/leebyron/ecmascript-export-default-from